content/_releases/22/52.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Product Update - v22.52
+description: Details of the product update v22.52 from 29th of December 2022.
+layout: rel
+version: "22.52"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-12-29
+redirect_from:
+  - /releases/22.52.html
+---
+
+## Features
+
+### Additing tenant roles
+
+We have changed the name of permission `tenants.tenant.edit_roles` to `global.tenant.edit_roles`.
+
+Now `PATCH /v2/tenants/:tenantId/roles` allows to create a new role with permissions in a tenant scope. We have also updatet API endpoint `PATCH /v2/tenants/{tenant_id}/members/{user_id}` which allows to grant any role from tenants scope to any tenant user. This will allow to give any user `tenants.user.generate_one_time_token` permission.
```

Comment:
We have also updated API endpoint `PATCH /v2/tenants/{tenant_id}/members/{user_id}` which allows granting any role from the tenant's scope to any user. With this endpoint the permission `tenants.user.generate_one_time_token` can be granted to any user inside the tenant.